runtime/legion/legion_ops.cc

```diff
@@ -2365,6 +2411,8 @@ namespace Legion {
       if (remap_region)
       {
         region.impl->get_references(mapped_instances);
+        // TODO: Implement physical tracing for map operation
```

Comment:
Let's aggregate the TODOs as github issue for a feature enhancement for physical tracing.

Reply:
Ping on this again to make sure it gets done.